NEWS.md

```diff
@@ -5,6 +5,16 @@ NEW FUNCTIONS
 * `logSafe()` to compute logarithm of values that could be close to 0 or slightly 
 negative.
 
+* `foldSafe()` to compute `x / y` when `y` could be negative. All values below a 
```

Comment:
"when x or y could be negative or zero"?

Reply:
True, added.